src/commandsAndMenu.tsx

```diff
@@ -410,6 +410,58 @@ export function addCommands(
     }
   });
 
+  /** Add git reset --hard <remote-tracking-branch> command */
+  commands.addCommand(CommandIDs.gitResetToRemote, {
+    label: trans.__('Reset to Remote'),
+    caption: trans.__('Reset Current Branch to Remote State'),
+    isEnabled: () => gitModel.pathRepository !== null,
+    execute: async _ => {
```

Comment:
Prefer `()` to `_` as the latter is a valid variable name when the former explicitly state no arguments are expected.

```suggestion
    execute: async () => {
```